lib/wise_homex/models/account.ex

```diff
@@ -7,6 +7,7 @@ defmodule WiseHomex.Account do
     belongs_to :owner, __MODULE__
     has_many :admin_integrations, WiseHomex.AdminIntegration, foreign_key: :admin_id
     has_many :account_users, WiseHomex.AccountUser
+    has_many :permanent_account_keys, WiseHomex.PermanentAccountKeys
```

Comment:
This relation is not exposed by the API, so it should probably be deleted here.

Reply:
Removed :+1: (I just added so that the association was complete) :sunny: